src/time.cr

```diff
@@ -690,10 +707,10 @@ struct Time
     Span.new(nanoseconds: NANOSECONDS_PER_SECOND * (offset_seconds % SECONDS_PER_DAY) + nanosecond)
   end
 
-  # Returns the day of the week (`Sunday..Saturday`).
+  # Returns the day of the week (`Monday..Sunday`).
   def day_of_week : Time::DayOfWeek
-    value = ((offset_seconds / SECONDS_PER_DAY) + 1) % 7
-    DayOfWeek.new value.to_i
+    days = offset_seconds / SECONDS_PER_DAY
+    DayOfWeek.new days.to_i % 7 + 1
```

Comment:
Do not want sound nit-picking, but I was thinking if would make sense to break it in two methods, one private to return the weekday :

```crystal
def day_of_week : Time::DayOfWeek
  DayOfWeek.new(weekday_number)
end

private def weekday_number
  days = offset_seconds / SECONDS_PER_DAY
  days.to_i % 7 + 1
end
```

Reply:
What would be the benefit of this additional method?

Reply:
The benefit I see is readability because we do two arithmetical operations to get the number of the day of the week. 

From the perspective who is reading the code, to read `DayOfWeek.new(weekday_number)` instead of `DayOfWeek.new(days.to_i % 7 + 1)` becomes more self explanatory.

Reply:
Hm, then fitting in an appropriately named variable name would do the same result.